src/lib/ListMetrics.js

```diff
@@ -225,6 +226,9 @@ export default class ListMetrics {
                 } else {
                     let val = parseInt(item[1], 10);
                     val = isNaN(val) ? 0 : val;
+                    if (val < 0) {
+                        isAnyMetricNegative = true;
```

Comment:
You can use `Array.prototype.every` to exit early when any metric is negative.

Reply:
Updated.